fury/shaders/tests/test_base.py

```diff
@@ -350,6 +351,7 @@ def test_load():
     npt.assert_string_equal(load(dummy_file_name), dummy_file_contents)
 
     os.remove(os.path.join(SHADERS_DIR, dummy_file_name))
+"""
```

Comment:
remove the double quote, quit the test, and check if `assert_warns` works as expected

Reply:
By quitting the test, do you mean remove the current test and leave just the `assert_warns`?